cmake/ctest/drivers/atdm/cee-rhel6/local-driver.sh

```diff
@@ -1,7 +1,14 @@
 #!/bin/bash -l
 
-if [ "${Trilinos_CTEST_DO_ALL_AT_ONCE}" == "" ] ; then
-  export Trilinos_CTEST_DO_ALL_AT_ONCE=TRUE
+set +x
+
+# Need to load env so we define some vars
+source $WORKSPACE/Trilinos/cmake/std/atdm/load-env.sh $JOB_NAME
+
+# Make adjustments for mini build of Trilinos for SPARC
+if atdm_match_buildname_keyword mini ; then
+  echo "This is a mini build of Trilinos for SPARC!"
+  export ATDM_CONFIG_CONFIGURE_OPTIONS_FILES=cmake/std/atdm/ATDMDevEnv.cmake,cmake/std/atdm/apps/sparc/SPARC_MiniTrilinos_PACKAGES.cmake
```

Comment:
@bartlettroscoe: Can we remove `SPARC_MiniTrilinos_PACKAGES.cmake` here and use `SPARCMiniTrilinosPackagesEnables.cmake` instead?

Reply:
@e10harvey, technically, `Trilinos_PACKAGES` is only correctly understood by the `tribits_ctest_driver()` and not the inner configure.  That is why there are two separate `*.cmake` files.

Reply:
@bartlettroscoe, thank you. Is this configuration change getting picked up by the `tribits_ctest_driver`? I ask because I do not see evidence of this in `Configure.xml`.

Reply:
@e10harvey, you should see evidence f that in the the smart-jenkins-driver.out file.  Can you attach that file here?

Reply:
@bartlettroscoe, please see atdv-352.